contribution.md

```diff
@@ -0,0 +1,14 @@
+## Contributing to this Repository
+Contributors are welcome. To contribute
+- fork the project
+- clone it using
+```
+git clone <your forked repo URL>
+```
+- Open index.html
+- Enhance the stuff requested.
+
+```
+<br>
+## Hacktoberfest 
+PR's are welcome. Lets do it in good spirit. Claim the issue you want to work on.
```

Comment:
Hi, a more relevant file has been added instead. So please add a link to that file in the readme for new contributor :)